extension/tests/component/__snapshots__/createOrder.test.ts.snap

```diff
@@ -8,7 +8,7 @@ exports[`Create Order Happy Path Should return 201 when mollie order created fro
     "createdAt": "2021-12-21T07:33:07+00:00",
     "id": "b2bd1698-9923-4704-9729-02db2de495d1",
     "request": "{"cartId":"84e5f274-c413-404e-b938-ffb45ce63f61","transactionId":"2b5f68ad-ae94-4bf1-ae41-7096e5142f89","createPayment":{}}",
-    "response": "{"mollieOrderId":"ord_ca1j7q","checkoutUrl":"https://www.mollie.com/checkout/order/ca1j7q","transactionId":"2b5f68ad-ae94-4bf1-ae41-7096e5142f89"}",
+    "response": "{"mollieOrderId":"ord_1.ca1j7q","checkoutUrl":"https://www.mollie.com/checkout/order/ca1j7q","transactionId":"2b5f68ad-ae94-4bf1-ae41-7096e5142f89"}",
```

Comment:
The `checkoutUrl` should also represent the new `mollieOrderId`, right?

Reply:
Not needed